dg/media/coco/app/scripts/auth_offline_backend.js

```diff
@@ -26,13 +26,13 @@ define([
   }
   
   // if u, p matches that in user table, sets login state = true 
-  var login = function(username, password, language, partner_name){
+  var login = function(username, password, language, partner_name, partner_id, user_id){
```

Comment:
We were using partner name earlier for rendering various fields. If i would have not taken i have to query on backend to get id and need to change existing field changes based on partner. To avoid all these i have done that.